docs/snippets/quickstart-install.md

```diff
@@ -112,21 +114,29 @@ To get a local deployment of Knative, run the `quickstart` plugin:
 
     1. Install Knative and Kubernetes in a minikube instance by running:
 
+        !!! note
+            The minikube cluster will be created with 6&nbsp;GB of RAM. If you don't have enough memory, you can change to a
+            different value not lower than 3&nbsp;GB by running the command `minikube config set memory 3078` before this command.
         ```bash
         kn quickstart minikube
         ```
 
-    1. After the plugin is finished, verify you have a cluster called `knative`:
-
-        ```bash
-        minikube profile list
-        ```
-    1. To finish setting up networking for minikube, start the `minikube tunnel` process in a separate terminal window:
+    1. The output of the previous command asked you to run minikube tunnel.
+       Run the following command to start the process in a secondary terminal window, then return to the primary window and press enter to continue:
         ```bash
         minikube tunnel --profile knative
         ```
-        The tunnel must continue to run in a terminal window while you are using your Knative `quickstart` environment.
+        The tunnel must continue to run in a terminal window while you are using your Knative `quickstart` environment at any time.
+
+        The tunnel command is required as it would allow Knative ingress service to be accesible as a LoadBalancer from your host computer.
```

Comment:
```suggestion
        The tunnel command is required because it allows your cluster to access Knative ingress service as a LoadBalancer from your host computer.
```